src/actions/index.js

```diff
@@ -42,10 +57,27 @@ const payload = {
             }
         }
     },
+    /**
+     *
+     */
     getSelectComponent: (id) => {
         return {
             id
         }
+    },
+    /**
+     *
+     */
```

Comment:
All of these need proper documentation.  If your linting isn't failing on these kinds of comments it should be.